java/test/jmri/jmrit/operations/trains/XmlTest.java

```diff
@@ -26,6 +28,68 @@
  * @version $Revision$
  */
 public class XmlTest extends OperationsTestCase {
+    
+    public void testFilePathNames() {
+        // test the build report path name
+        Assert.assertEquals("buildstatus", TrainManagerXml.BUILD_STATUS);
+        Assert.assertEquals(OperationsXml.getFileLocation() +
+                "operations" +
+                File.separator +
+                "JUnitTest" +
+                File.separator +
+                "buildstatus" +
```

Comment:
These strings can be constants. Given that a constant is unchanging, there is no advantage to not using the constant.